# Autol LiDAR packet.
uint8[1434] data	# packet data
